demo/src/examples/load-media-form-component.js

```diff
@@ -114,12 +115,35 @@ export class LoadMediaFormComponent extends LitElement {
     `;
   }
 
+  updated(_changedProperties) {
+    super.updated(_changedProperties);
+
+    if (_changedProperties.has('drmSettingsShown') && this.drmSettingsShown) {
+      this.shadowRoot.querySelector('.drm-settings-container').classList.toggle('active', true);
+    }
+  }
+
+  #onFormAnimationEnd(e) {
+    if (e.target.classList.contains('shrink')) {
+      e.target.classList.toggle('active', false);
```

Comment:
In this case this might be easier to read
```suggestion
      e.target.classList.remove('active');
```

Reply:
Done !